spark_log_parser/loaders/__init__.py

```diff
@@ -12,14 +12,16 @@
 from aiodataloader import DataLoader
 from stream_unzip import stream_unzip
 
+logger = logging.getLogger("Loaders")
+
 FILE_SKIP_PATTERNS = [".DS_Store".lower(), "__MACOSX".lower(), "/."]
 
-logger = logging.getLogger("Loaders")
+DEFAULT_MAX_FILESIZE = 5000000000
```

Comment:
I pulled this out into a module global because I can't reference `ArchiveExtractionThresholds.size` on this line - https://github.com/synccomputingcode/spark_log_parser/pull/39/files#diff-e6eadcc243cd6e0abd0b257ed701c0e4b5a24d8712a487a823c9f3a929f561c3R51